docs/user-guide/creating-application/job-and-cronjob.md

```diff
@@ -1,5 +1,10 @@
 ## Job and Cronjob chart
 
+To select job and cronjob first we have to select the deployment templet . Then after select the dropdown button in which we can select the job and cronjob instead of rollout templet. then after we can continue as mention below .
```

Comment:
Too many errors grammatical errors, please get it reviewed by @arushi09-hub once